industrial_taxonomy/pipeline/README.md

```diff
@@ -1,5 +1,18 @@
 # Steps in data pipeline
 
+:warning: When testing omit the `--production` flag from flow commands
+
 ## 1. Collect data
 
 Run `python industrial_taxonomy/pipeline/sic/flow.py --production run` to create the SIC code - name lookups
+
+## 2. Fuzzy-match Glass organisations to Companies House companies
+
+```bash
+python industrial_taxonomy/pipeline/glass_house/flow.py \
+ --environment=conda \
+ --production \
+ run \
+ --clean-names true \
+ --with batch
```

Comment:
Is this a standard `FlowSpec` parameter? Asking because it doesn't seem that intuitive as a name

Reply:
Which parameter? `--with batch` ?
If so, yes: https://docs.metaflow.org/metaflow/scaling#using-aws-batch

Either way: what's not intuitive?

Reply:
I was going to say `--batch true` might be more straightforward, but now see that you can pass in an extended set of comma separated args with the specifics

Reply:
Yes, and `--with` is a general pattern to apply decorators from the command-line

E.g. `--with deco` is (almost) the same as adding `@deco` to every flow step.